### PR TITLE
Update Items page

### PR DESCRIPTION
**Pull request by hbeachey**
- Clean up column names/data
- Add Sort functionality
- Add Filter functionality
- Put item actions into a collapsable menu

Satisfies LW-51 and LW-52, styling cleanup might be wanted.

Note: Renew is commented out until we figure out how to make it actually work.

### CODE REVIEW

File: src/components/LoanResources/ResourceList/Resource/Actions/presenter.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  //     return (<span>{message}</span>)
+  //   } else {
+  //     return (<button onClick={onRenewClick}>Renew</button>)
+  //   }
+  // } else {
+  //   return null
+  // }
+}
+
+export const hasActions = (item) => {
```
Comment (jgondron) [inline src/components/LoanResources/ResourceList/Resource/Actions/presenter.js]: This doesn't appear to be used.

File: src/components/LoanResources/ResourceList/Resource/presenter.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { hasActions } from './Actions/presenter'
+
+const Card = (className, prefix, data) => {
+  if (data) {
+    return (<div className={className}><span>{prefix + data}</span></div>)
+  }
+  return <div className={className} />
+}
+
+const dueDate = (item, renewal) => {
```
Comment (jgondron) [inline src/components/LoanResources/ResourceList/Resource/presenter.js]: This should also be done in parent and passed in as props

Comment (hbeachey) [inline src/components/LoanResources/ResourceList/Resource/presenter.js]: Perhaps it'd be smarter to just merge the "renew" pr into this branch, as some of these suggestions were already done there.

File: src/components/LoanResources/ResourceList/Resource/Actions/presenter.js (hunk 1 above)
Comment (hbeachey) [inline src/components/LoanResources/ResourceList/Resource/Actions/presenter.js]: This is used in "ResourceList/Resource/presenter.js"

**Pull request merged**